tutorials/simulators/1_aer_provider.ipynb

```diff
@@ -367,7 +367,8 @@
     "| `save_unitary` | Save the simulator state as unitary matrix of the run circuit | `\"automatic\"`, `\"unitary\"`|\n",
     "| `save_superop` | Save the simulator state as superoperator matrix of the run circuit | `\"automatic\"`, `\"superop\"`|\n",
     "\n",
-    "Note that these instructions are only supported by the Aer simulator and will result in an error if a circuit containing them is run on a non-simulator backend such as an IBM Quantum device."
+    "Note: These instructions are only supported by the Aer simulator and will result in an error if a circuit containing them is run on a non-simulator backend such as an IBM Quantum device."
+    "Note: The following must be imported to save the state of the simulator 'from qiskit.providers.aer import Aer'.
```

Comment:
Actually besides the import path issue I mentioned before (which I missed here because of syntax highlighting) this also isn't actually 100% accurate. Any thing from `qiskit_aer` can be imported to add the monkey patched methods to `QuantumCircuit`.